contracts/RelayHub.sol

```diff
@@ -143,7 +144,7 @@ contract RelayHub is IRelayHub {
         require(relays[relay].owner == msg.sender, "not owner");
         relays[relay].unstakeTime = relays[relay].unstakeDelay + now;
         // Start the unstake counter
-        if (relays[relay].state != State.PENALIZED) {
+        if (relays[relay].state != State.PENALIZED && relays[relay].state != State.REMOVED) {
```

Comment:
Not sure what the `relays[relay].state != State.REMOVED` check achieves.

Reply:
Without it, calling remove again will re-trigger the unstake timeout...

Reply:
unstake timeout is set right before this `if`, this branch merely  sets `relays[relay].state = State.REMOVED`.